msg/AutotuneAttitudeControlStatus.msg

```diff
@@ -1,8 +1,10 @@
-uint64 timestamp                # time since system start (microseconds)
+# Autotune attitude control status
```

Comment:
@mahima-yoga While we're here can we add documentation in the standard form for this message please? After all, you're now the expert and no one will be able to do it better.

I have started it - for inspiration see https://docs.px4.io/main/en/msg_docs/ArmingCheckReply.html

I'd hope for a more complete description below this one explaining what publishes and subscribes it, and anything that can't be inferred from the name that would help developers maintain it and know when they should touch it.